configs/components/openssl-1.1.1.rb

```diff
@@ -74,6 +74,8 @@
       sslflags = '386'
     elsif platform.architecture =~ /aarch64$/
       target = 'linux-aarch64'
+    elsif platform.architecture =~ /ppc64le|ppc64el/ # Little-endian
+      target = 'linux-ppc64le'
```

Comment:
Won't this impact the other ppc64 platforms, that are cross compiled?

Reply:
Cross compiled platforms should be caught on line 29